.asf.yaml

```diff
@@ -77,11 +77,31 @@ github:
     branch-2.4: {}
     branch-2.5: {}
     branch-2.6: {}
-    branch-2.7: {}
-    branch-2.8: {}
-    branch-2.9: {}
-    branch-2.10: {}
-    branch-2.11: {}
+    branch-2.7:
+      required_pull_request_reviews:
+        dismiss_stale_reviews: false
+        require_code_owner_reviews: true
```

Comment:
```suggestion
```

ditto others.

We don't have CODEOWNER and `dismiss_stale_reviews: false` is the default.

Reply:
Also I think 2.7 should be end of life once 2.11 released?

Reply:
> Also I think 2.7 should be end of life once 2.11 released?

Maybe we will not have a release for branch-2.7. But the PR cherry-picking can also happen on branch-2.7. As I know, there are many users still using 2.7.x

Reply:
@tisonkun addressed. Please check again.